sechub-server/src/main/java/com/daimler/sechub/server/SecHubSystemPropertyInjector.java

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+package com.daimler.sechub.server;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.daimler.sechub.sharedkernel.MustBeDocumented;
+
+/**
+ * This component injects some special SecHub Spring Boot values into
+ * corresponding system properties. So we can configure those parts in our
+ * `application.yaml` file.
+ * 
+ * @author Albert Tregnaghi
+ *
+ */
+@Component
+public class SecHubSystemPropertyInjector {
+
+    // see https://github.com/Daimler/sechub/issues/68 for details
```

Comment:
This comments does not lead to anything related to Diffie-Hellman or key length. Please remove it.

Reply:
will remove both (pds + sechub server)

Reply:
done